lib/samples/find_address_with_reverse_geocode/find_address_with_reverse_geocode_sample.dart

```diff
@@ -47,158 +51,78 @@ class _FindAddressWithReverseGeocodeSampleState
-        value: _worldLocatorTask,
-        groupValue: _locatorTask,
-        onChanged: onServiceChanged,
-      ),
-      RadioListTile<LocatorTask>(
-        title: const Text('San Diego'),
-        value: _sanDiegoLocatorTask,
-        groupValue: _locatorTask,
-        onChanged: onServiceChanged,
-      ),
-    ]);
-  }
-
-  Widget buildSearchControls(BuildContext context) {
-    return Row(
-      children: [
-        Expanded(
-          child: Container(
-            padding: const EdgeInsets.symmetric(horizontal: 10),
-            color: Colors.white,
-            child: TextField(
-              decoration: const InputDecoration(
-                hintText: 'Search...',
-              ),
-              onSubmitted: onSearchSubmitted,
-            ),
-          ),
-        ),
-        IconButton(
-          icon: const Icon(
-            Icons.list,
-            color: Colors.white,
-          ),
-          onPressed: () => _scaffoldStateKey.currentState!.openEndDrawer(),
-        ),
-      ],
-    );
-  }
-
   void onMapViewReady() async {
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISImagery);
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
 
     final image = await ArcGISImage.fromAsset('assets/pin_circle_red.png');
     final pictureMarkerSymbol = PictureMarkerSymbol.withImage(image);
     pictureMarkerSymbol.width = 35;
     pictureMarkerSymbol.height = 35;
     pictureMarkerSymbol.offsetY = pictureMarkerSymbol.height / 2;
+
     _graphicsOverlay.renderer = SimpleRenderer(symbol: pictureMarkerSymbol);
-    _graphicsOverlay.graphics.add(_graphic);
     _mapViewController.graphicsOverlays.add(_graphicsOverlay);
 
     _mapViewController.arcGISMap = map;
-
-    _geocodeParameters.minScore = 75;
-    _geocodeParameters.resultAttributeNames
-        .addAll(['Place_addr', 'Match_addr']);
+    _mapViewController.setViewpoint(_initialViewpoint);
 
     await Future.wait([
       _worldLocatorTask.load(),
-      _sanDiegoLocatorTask.load(),
     ]);
 
     setState(() => _ready = true);
   }
 
   void onTap(Offset localPosition) async {
-    final identifyGraphicsOverlayResult =
-        await _mapViewController.identifyGraphicsOverlay(
-      _graphicsOverlay,
-      screenPoint: localPosition,
-      tolerance: 5,
-    );
+    if (_graphicsOverlay.graphics.isNotEmpty) _graphicsOverlay.graphics.clear();
 
-    if (identifyGraphicsOverlayResult.graphics.isEmpty) return;
+    final mapTapPoint =
+        _mapViewController.screenToLocation(screen: localPosition);
+    if (mapTapPoint == null) return;
 
-    if (mounted) {
-      final graphic = identifyGraphicsOverlayResult.graphics.first;
-      final center = graphic.geometry!.extent.center;
-      final x = center.x.toStringAsFixed(3);
-      final y = center.y.toStringAsFixed(3);
-      final matchAddr = graphic.attributes['Match_addr'];
+    final normalizedTapPoint =
+        GeometryEngine.normalizeCentralMeridian(geometry: mapTapPoint);
```

Comment:
I see that `normalizeCentralMeridian` in the reference sample code, but I'm not sure it's necessary. I think it might just be adding extra complication. @jenmerritt  do you know if we need to be doing this?

Reply:
Because we have wraparoundmode enabled we should normalize before passing to the geocode operation as if you tap a point across the date line it messes things up. We could remove this and set `WrapAroundMode.disabled` on the mapview instead. But given other implementations are using this I think we should follow suit.

One thing we need to do in the sample (which I'll go through with you on our call @shokhinaToj ) is to add comments in the code, so we can add a little note on this too.

Reply:
Oh, I though WrapAroundMode was disabled by default. Everything I said after that was backwards, don't mind me.

But! I do have one other actually good point: `normalizedTapPoint` might be `null`, so instead of `!` maybe it should do:
```dart
if (normalizedTapPoint == null) return;
```
Seeing a `!` is always a little scary unless we can directly reason that it can't be null. Here we're relying on the internal machinations of `GeometryEngine`, so we shouldn't assume too much.

Reply:
Gotcha! I will make sure to apply this change